sdk/azcore/internal/pollers/util.go

```diff
@@ -23,6 +23,7 @@ import (
 
 // the well-known set of LRO status/provisioning state values.
 const (
+	StatusCompleted  = "Completed"
```

Comment:
While I'm not opposed to adding this, I believe this is in violation of the [ARM RPC spec](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#provisioningstate-property).  Let me follow up on that.

Reply:
Thanks, I was unaware of the spec. Just encountered the bug and tried to solve it this way :) . Let me know if there is a more elegant solution.

Reply:
Hi @jhendrixMSFT , any update on this? 

If there is a better way we can fix the issue please let me know.

Reply:
Sorry for the delay, I just pinged the owners again to get their input on this.

Reply:
The service is in error here, it shouldn't be returned `Completed`.  I'm following up with the service team to get this fixed.